CHANGELOG.md

```diff
@@ -97,9 +97,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - move parameters from `otelagent.enabled` to `otelcolInstrumentation.enabled`
   - move `otelagent.daemonset.nodeSelector` to `otelcolInstrumentation.statefulset.nodeSelector`
   - move `otelagent.daemonset.priorityClassName` to `otelcolInstrumentation.statefulset.priorityClassName`
-  - move `otelcol.deployment.replicas` to `tracesGateway.deployment.replicas`
-  - move `otelcol.deployment.nodeSelector` to `tracesGateway.deployment.nodeSelector`
-  - move `otelcol.deployment.priorityClassName` to `tracesGateway.deployment.priorityClassName`
+  - move `otelcol.deployment.replicas` to `tracesSampler.deployment.replicas`
+  - move `otelcol.deployment.nodeSelector` to `tracesSampler.deployment.nodeSelector`
+  - move `otelcol.deployment.priorityClassName` to `tracesSampler.deployment.priorityClassName`
```

Comment:
Can't we just say the same thing here as we do in the migration doc, that is: `- Moved parameters from otelagent.* to otelcolInstrumentation.*?`